src/contrib/cublas/cublas_utils.h

```diff
@@ -0,0 +1,53 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file Use external cudnn utils function
+ */
+
+#ifndef TVM_CONTRIB_CUBLAS_CUBLAS_UTILS_H_
+#define TVM_CONTRIB_CUBLAS_CUBLAS_UTILS_H_
+
+#include <dmlc/logging.h>
+
+extern "C" {
+#include <cublas_v2.h>
+}
+
+namespace tvm {
+namespace contrib {
+
+inline const char* get_cublas_error_string(int error) {
```

Comment:
+1 for handling cublas error string with inline function instead of macro.

Reply:
However, can we change the name of the function using camel cases, following [Google C/C++ style](https://google.github.io/styleguide/cppguide.html#Function_Names) ?